### PR TITLE
feat: add alma8 and alma9 packer configs

### PR DESCRIPTION
**Pull request by lleyton**
These are mainly just copies of the Rocky configs, but with the Alma repos instead.

note: haven't personally tested these, yet

### CODE REVIEW

File: alma9/Makefile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.PHONY: all clean
+
+all: alma9.tar.gz
+
+$(eval $(call check_packages_deps))
+
+alma9.tar.gz: check-deps clean http/alma.ks
+	${PACKER} init alma9.pkr.hcl && ${PACKER} build alma9.pkr.hcl
+
+http/alma.ks: http/alma.ks.in
```
Comment (alexsander-souza) [inline alma9/Makefile:19]: remove this

File: alma9/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+You can customize the deployment image by modifying http/alma.ks. See the [RHEL kickstart documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_installation/kickstart-commands-and-options-reference_installing-rhel-as-an-experienced-user#part-or-partition_kickstart-commands-for-handling-storage) for more information.
+
+## Building the image using a proxy
+
+The Packer template downloads the Alma ISO image from the Internet. You can tell Packer to use a proxy by setting the HTTP_PROXY environment variable to point to your proxy server. You can also  redefine alma_iso_url to a local file. If you want to skip the base image integrity check, set iso_checksum_type to none and remove iso_checksum.
+
+To use a proxy during the installation define the `KS_PROXY` variable in the environment, as bellow:
+
+```shell
+export KS_PROXY="--proxy=\"${HTTP_PROXY}\""
```
Comment (alexsander-souza) [inline alma9/README.md:29]: update to new format

File: alma9/alma9.pkr.hcl [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+variable "alma_iso_url" {
+  type    = string
+  default = "https://repo.almalinux.org/almalinux/9/isos/x86_64/AlmaLinux-9-latest-x86_64-boot.iso"
+}
+
+variable "alma_sha256sum_url" {
+  type    = string
+  default = "https://repo.almalinux.org/almalinux/9/isos/x86_64/CHECKSUM"
+}
+
```
Comment (alexsander-souza) [inline alma9/alma9.pkr.hcl:32]: add variables for mirrors

File: alma9/alma9.pkr.hcl [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  boot_wait        = "3s"
+  communicator     = "none"
+  disk_size        = "4G"
+  headless         = var.headless
+  http_directory   = "http"
+  iso_checksum     = "file:${var.alma_sha256sum_url}"
+  iso_url          = "${var.alma_iso_url}"
+  memory           = 2048
+  qemuargs         = [["-serial", "stdio"], ["-cpu", "host"]]
+  shutdown_timeout = "1h"
```
Comment (alexsander-souza) [inline alma9/alma9.pkr.hcl:44]: add template expansion

File: alma9/http/alma.ks.in
Comment (alexsander-souza) [inline alma9/http/alma.ks.in:1]: convert to packer template file

**Review by alexsander-souza — CHANGES_REQUESTED** [review]
awesome work, but the alma9 template need some changes.

basically just do the same changes done to alma8 to use the packer template file format and variable expansion.

**Comment by lleyton** [comment]
Just impled the suggestions, let me know if it looks good now

**Comment by SK1Y101** [comment]
Hi @lleyton, just hoping in to confirm that your alma9 changes allow the image to be built with the proxy settings as expected with the current packer version as expected!

**Review by alexsander-souza — APPROVED** [review]
all good, lgtm

**Pull request merged**